CONTRIBUTING.md

```diff
@@ -75,18 +75,30 @@ $ git config user.email "my-email@example.com"
 
 ## Create a new branch
 
-Do not develop on the `master` branch: feature branches are intended to be
+First, you need to decide which release branch (e.g., `master`, `0.2`) to
+create the feature branch from. If you intend to add a new feature, then
+`master` is the right branch. Bug fixes however should also be applied to
+other releases, so you should create your feature branch from the release
```

Comment:
`create your feature branch` - I think it should be `create your bugfix branch` or something like that because it will be not a feature branch but a bug fix branch.

Reply:
I think feature branch is actually the right term here. It's quite commonly used for branches that are only used to implement a single ticket (see for example the [Atlassian docs](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) or the [GitLab docs](https://docs.gitlab.com/ee/workflow/workflow.html)). A bugfix isn't strictly a feature, but I think that it's better to say that a bugfix is implemented in a feature branch then that a feature is implemented in a bugfix branch as feature seems to me to be the more general term here.

Reply:
You convinced me. I am OK with `feature branch`.